apps/observability/otel-collector/otel-collector-config.yaml

```diff
@@ -18,7 +18,7 @@ exporters:
     endpoint: 0.0.0.0:8510
 
   otlp/tempo:
```

Comment:
There isn't a need at this moment, however, when/if we send data to a backend that requires a security token to be passed in this way of specifying the configuration will not work (As we do not want to expose the tokens).

It may be prudent to determine a way to inject the config.yaml without seeing it's content to support this ahead of time.

An example for signoz: https://signoz.io/opentelemetry/collector-nodejs/#creating-the-collector-configuration-file

In the example it has a section to add a header for all out-bound requests:
```
exporters:

  otlp:
    endpoint: ${OTEL_EXPORTER_OTLP_ENDPOINT}
    tls:
      insecure: false
    headers:
      "signoz-ingestion-key": ${SIGNOZ_INGESTION_KEY}
  debug:
    verbosity: normal

```